src/coreclr/jit/emitarm64.cpp

```diff
@@ -24825,6 +25090,17 @@ void emitter::emitDispInsHelp(
             emitDispReg(id->idReg3(), id->idOpSize(), false);                                   // mmmmm
             break;
 
+        // <Pd>, <Pn>, <Pm>.<T>[<Wv>, <imm>]
+        case IF_SVE_DV_4A: // ........ix.xxxvv ..NNNN.MMMM.DDDD -- SVE broadcast predicate element
+            emitDispPredicateReg(id->idReg1(), PREDICATE_NONE, id->idInsOpt(), true);   // DDDD
```

Comment:
These should use the get predicate function instead of `PREDICATE_*`

Reply:
Got it, fixed.